common/test/acceptance/tests/lms/test_lms_user_preview.py

```diff
@@ -12,6 +13,72 @@
 from textwrap import dedent
 
 
+class StaffViewEnrollmentButton(UniqueCourseTest):
+    """Tests for the global staff only enrollment button"""
+
+    USERNAME = "STAFF_TESTER"
+    EMAIL = "johndoe@example.com"
+
+    def setUp(self):
+        super(StaffViewEnrollmentButton, self).setUp()
+
+        self.course_info_page = CourseInfoPage(self.browser, self.course_id)
+
+        # Install a course with sections/problems, tabs, updates, and handouts
+        self.course_fixture = CourseFixture(
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run'],
+            self.course_info['display_name'],
+        )
+
+        self.course_fixture.install()
+
+    def test_register_for_course_with_button(self):
```

Comment:
I would prefer to see some docstrings explaining the test points (we typically use BDD-style comments).